src/RowItem/Labels.js

```diff
@@ -37,6 +40,27 @@ function Labels(props) {
   let annotationsSVG = [];
   let rowCenter = rowLength / 2;
   let iTree = new IntervalTree(rowCenter);
+
+  annotationRanges = values(
+    reduce(
+      annotationRanges,
+      (accum, annotationRange) => {
+        if (
+          annotationRange.annotation.annotationTypePlural === "parts" ||
+          !accum[annotationRange.id] ||
```

Comment:
We'll want to prefix the id with the annotationTypePlural so it looks like 
```
!accum[
            annotationRange.annotation.annotationTypePlural +
              "--" +
              annotationRange.id
          ]
``` 

kind of verbose but necessary if id's are re-used across annotation types